### PR TITLE
Create jambalaya.tex

### PR DESCRIPTION
**Pull request by MaxwellBo**

### CODE REVIEW

**Comment by TRManderson** [comment]
Can you add an `\input{recipes/jambalaya.tex}` to `main.tex`?

File: recipes/jambalaya.tex [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+\Ing{1 & 1\fr2 cayenne pepper, extra hot}
+\Ing{ 2 chopped and seeded Bird's eye chillies}
+\Ing{1 green capsicum, diced}
+\Ing{4 celery stalks, chopped}
+Fry down the onions, garlic, herbs, and all of the root vegetables including celery and capsicum, till translucent
+\Ing{2 chicken breasts}
+Fry off chorizo and chicken until browned
+\Ing{400g of chicken stock}
+\Ing{1 can of crushed tomatoes (optional)}
+\Ing{400g of rice}
```
Comment (TRManderson) [inline recipes/jambalaya.tex]: pls specify cooked/uncooked

Comment (MaxwellBo) [inline recipes/jambalaya.tex]: I would assume uncooked

Comment (TRManderson) [inline recipes/jambalaya.tex]: Haven't you cooked this before?

Comment (MaxwellBo) [inline recipes/jambalaya.tex]: I had help

Comment (MaxwellBo) [inline recipes/jambalaya.tex]: It's nice though

**Pull request merged**